tools/sggolangcilint/golangci.yml

```diff
@@ -160,120 +172,109 @@ linters:
-    # [fast: true, auto-fix: false]
-    - gochecknoglobals
-
-    # Inspect source code for security problems.
-    # [fast: false, auto-fix: false]
-    - gosec
-
-    # Find sending http request without context.Context.
-    # [fast: false, auto-fix: false]
-    - noctx
-
-    # Drop-in replacement of golint.
-    # [fast: false, auto-fix: false]
-    - revive
-
-    # Check for pass []any as any in variadic func(...any).
-    # [fast: false, auto-fix: false]
-    - asasalint
-
-    # Check for dangerous unicode character sequences.
-    # [fast: true, auto-fix: false]
-    - bidichk
-
-    # Check types passed to the json encoding functions.
-    # [fast: false, auto-fix: false]
-    - errchkjson
-
-    # Check for the possibility to use variables/constants from the Go standard library.
-    # [fast: true, auto-fix: false]
-    - usestdlibvars
-
-    # Checks for mistakes with OpenTelemetry/Census spans.
-    # [fast: false, auto-fix: false]
-    - spancheck
-
-    # Ensure consistent code style when using log/slog.
-    - sloglint
-linters-settings:
-  gomodguard:
-    blocked:
-      modules:
-        - github.com/sirupsen/logrus:
-            recommendations:
-              - go.uber.org/zap
-            reason: "Use Zap for logging."
-        - github.com/rs/zerolog:
-            recommendations:
-              - go.uber.org/zap
-            reason: "Use Zap for logging."
-        - github.com/pkg/errors:
-            recommendations:
-              - errors
-              - fmt
-            reason: "Use the standard library error packages."
-        - golang.org/x/xerrors:
-            recommendations:
-              - errors
-              - fmt
-            reason: "Use the standard library error packages."
-        - github.com/golang/protobuf:
-            recommendations:
-              - google.golang.org/protobuf
-            reason: "The protobuf v1 module is deprecated."
-        - github.com/stretchr/testify:
-            recommendations:
-              - gotest.tools/v3
-            reason: "More reliably supports protobuf messages."
-
-  gofumpt:
-    extra-rules: true
-
-  gosec:
-    excludes:
-      # Flags for potentially-unsafe casting of ints. Prone to lots of false positives.
-      - G115
-
-  lll:
-    line-length: 120
-    tab-width: 1
-
-  misspell:
-    locale: US
-    ignore-words:
-      - cancelled
-      - cancelling
-      - analyses # allow analyses as a plural form of analysis
-
-  prealloc:
-    simple: true
-    range-loops: true
-    for-loops: true
-
-  protogetter:
-    skip-any-generated: true
-
-  staticcheck:
-    checks: [all]
-
-  stylecheck:
-    checks: [all]
-
-  errorlint:
-    # Don't enforce %w to minimize implicit interface leakage.
-    # https://github.com/dwmkerr/hacker-laws#hyrums-law-the-law-of-implicit-interfaces
-    errorf: false
-
-  gomoddirectives:
-    replace-local: true
+issues:
+  fix: false
+  exclude-rules:
+    - path: .sage
+      text: "(is a global variable|is unused)"
```

Comment:
The `--exclude` CLI argument was removed and we have to specify it here instead.

Reply:
I actually removed this, as I don't think we need it.